src/url-loader/fs-url-loader.ts

```diff
@@ -33,19 +31,11 @@ export class FSUrlLoader implements UrlLoader {
     this.root = root || '';
   }
 
-  canLoad(url: string): boolean {
-    const urlObject = parseUrl(url);
-    const pathname =
-        pathlib.normalize(decodeURIComponent(urlObject.pathname || ''));
-    return this._isValid(urlObject, pathname);
-  }
-
-  private _isValid(urlObject: Url, pathname: string) {
-    return (urlObject.protocol === 'file' || !urlObject.hostname) &&
-        !pathname.startsWith('../');
+  canLoad(url: ResolvedUrl): boolean {
+    return url.startsWith('file:///');
```

Comment:
We can parse the URL and check the scheme and host.

Reply:
We can, my thinking was just that this was quick, easy, and correct.